Student/Student/Submissions/SubmissionComments/SubmissionCommentAttemptCell.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class SubmissionCommentAttemptCell: UITableViewCell {
+    @IBOutlet weak var stackView: UIStackView?
+
+    var onFileTap: ((Submission?, File?) -> Void)?
+
+    func update(comment: SubmissionComment, submission: Submission?, onFileTap: @escaping (Submission?, File?) -> Void) {
+        accessibilityIdentifier = "SubmissionCommentsElement.attemptCell.\(comment.id)"
+        accessibilityLabel = String.localizedStringWithFormat(
+            NSLocalizedString("On %@ %@ submitted the following", bundle: .student, comment: ""),
+            DateFormatter.localizedString(from: comment.createdAt, dateStyle: .long, timeStyle: .short),
+            comment.authorName
+        )
+        self.onFileTap = onFileTap
+
+        for view in stackView?.arrangedSubviews ?? [] { view.removeFromSuperview() }
```

Comment:
At first glance I'm wondering if any of this is affecting scroll performance?  Is this cell scrolling choppy at all?  It may not be but something we may want to keep an eye on here.

Reply:
Maybe a little choppy here and there? If you know of a better way to dynamically replace a list of views in a reused cell, I'd love to make this smoother.